src/EditorFeatures/TestUtilities/Diagnostics/TestDiagnosticAnalyzerDriver.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace Microsoft.CodeAnalysis.UnitTests.Diagnostics
 {
-    public class TestDiagnosticAnalyzerDriver : IDisposable
+    public class TestDiagnosticAnalyzerDriver
```

Comment:
>public class TestDiagnosticAnalyzerDriver [](start = 4, length = 41)

Isn't removal of the interface a breaking change? The class is public, some clients could implicitly convert it to IDisposable, which is going to be an error now.